New extension manager (#12866)
jupyterlab/jupyterlab: docs/source/user/extensions.rst

.. _extension_manager:

You can use the Extension Manager in JupyterLab to manage extensions. That feature
may have been disabled by your system administrator for security reasons.

.. note::

>    Since JupyterLab v4, the default manager use PyPI.org as source for the available

Reviewer: ```suggestion
   Since JupyterLab v4, the default manager uses PyPI.org as source for the available
```

   extensions and ``pip`` to install them.

   An extension will be listed if the Python package has the classifier:
   `Framework :: Jupyter :: JupyterLab :: Extensions :: Prebuilt <https://pypi.org/search/?c=Framework+%3A%3A+Jupyter+%3A%3A+JupyterLab+%3A%3A+Extensions+%3A%3A+Prebuilt>`__
   Moreover it is advised for extension author to provide the ``install.json`` file

>    to specified the exact Python package name.

Reviewer: ```suggestion
   to specify the exact Python package name.
```

.. warning::

>    There are no verification that the extension is compatible with the current JupyterLab

Reviewer: ```suggestion
   There is no check to ensure the extension is compatible with the current JupyterLab
```

    avoid installing extensions you do not trust, and watch out for
    any extensions trying to masquerade as a trusted extension.

> The newly installed extension may require to restart JupyterLab.

Reviewer 2: ```suggestion
The newly installed extension may require JupyterLab to be restarted.
```
